aspnetcore/fundamentals/http-logging/samples/8.x/Program.cs

```diff
@@ -44,6 +44,8 @@
 #region snippet2
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddHttpLogging();
```

Comment:
The new overload `AddHttpLogging()` isn't available until net9.0.
```suggestion
builder.Services.AddHttpLogging(o => { });
```